fiftyone/operators/loader.py

```diff
@@ -89,11 +92,13 @@ def register_module(plugin_definition, mod):
     """
     try:
         pctx = PluginContext(plugin_definition)
-        KNOWN_PLUGIN_CONTEXTS[pctx.name] = pctx
+        if pctx.errors:
+            # Don't register if there are errors
+            return pctx
         mod.register(pctx)
```

Comment:
This should be called even with errors

Reply:
updated and put in `finally` so whether or not an error is raised, it gets added